docs/src/components/fields/RadioGroupOptionEditor.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import { FormControl, Checkbox } from 'react-bootstrap';
+import * as assign from 'object-assign';
+
+import { IFieldOptionEditorProps } from 'react-dynamic-formbuilder';
+
+interface IState { }
+
+
+export default class RadioGroupOptionEditor extends React.PureComponent<IFieldOptionEditorProps, IState> {
+
+    addOptionId: number;
+
+    constructor() {
+        super();
+
+        this.onLabelChange = this.onLabelChange.bind(this);
+        this.onRequiredChange = this.onRequiredChange.bind(this);
+        this.onLabelValueChange = this.onLabelValueChange.bind(this);
+        this.addOption = this.addOption.bind(this);
+        this.addOptionId = 0;
+    }
+
+    render() {
+        const { label, fields } = this.props.field;
+        const { required, unique } = this.props.field.options;
+        return (
+            <div>
+                <div>
+                    <span>Label</span>
+                    <FormControl type='text' value={label} onChange={this.onLabelChange} />
+                    <span>选项设置</span>
+                    {fields.map((item) => {
```

Comment:
Maybe we have a common control already, check how the dropdown field is implemented.